lib/project_types/script/messages/messages.rb

```diff
@@ -76,6 +76,19 @@ module Messages
 
           packages_outdated_cause: "The following npm packages are out of date: %s.",
           packages_outdated_help: "Update them by running {{cyan:npm install --save-dev %s}}.",
+
+          invalid_build_script: "Invalid build script.",
+          build_script_not_found: "Build script not found.",
+          # rubocop:disable Layout/LineLength
+          build_script_suggestion: "The root package.json should contain a script named build, which" \
+            "should rely on @shopify/scripts-toolchain-as to compile to WebAssembly.\n" \
+            "Example:\n" \
+            "build: npx shopify-scripts-toolchain-as build --src src/script.ts --binary <script_name>.wasm -- --lib node_modules --optimize --use Date=",
+
+          web_assembly_binary_not_found: "WebAssembly binary not found.",
+          web_assembly_binary_not_found_suggestion: "No WebAssembly binary found." \
+            "Make sure that your build npm script outputs the generated binary in the root of the directory." \
+            "The generated binary should match the script name: <script_name>.wasm",
         },
 
```

Comment:
Given that these are for internal use, I've just done a light edit.
>  packages_outdated_cause: "The following npm packages are out of date: %s.",
          packages_outdated_help: "Update them by running {{cyan:npm install --save-dev %s}}.",

 How about: 
`packages_outdated_cause: "These npm packages are out of date: %s",
     packages_outdated_help: "To update them, run: {{cyan:npm install --save-dev %s}}",
`
  

          invalid_build_script: "Invalid build script.",
          build_script_not_found: "Build script not found.",
          # rubocop:disable Layout/LineLength
          build_script_suggestion: "Root package.json needs a script named build, which" \
            "uses @shopify/scripts-toolchain-as to compile to WebAssembly.\n" \
            "Example:\n" \
            "build: npx shopify-scripts-toolchain-as build --src src/script.ts --binary <script_name>.wasm -- --lib node_modules --optimize --use Date=",

          web_assembly_binary_not_found: "WebAssembly binary not found.",
          web_assembly_binary_not_found_suggestion: "No WebAssembly binary found." \
            "Check that your build npm script outputs the generated binary to the root of the directory." \
            "Generated binary name needs to match the script name: <script_name>.wasm",
        },

Reply:
Make sense, updated!